src/libstd/prelude/mod.rs

```diff
@@ -11,8 +11,8 @@
 //! The Rust Prelude
 //!
 //! Because `std` is required by most serious Rust software, it is
-//! imported at the topmost level of every crate by default, as if the
-//! first line of each crate was
+//! imported at the topmost level of every crate by default, as if
+//! each crate contains the following:
```

Comment:
It's not even inserted at the top of the crate (check ` -Z unstable-options --pretty=expanded`). I think it does not to be mentioned where exactly it's inserted, as this is something of an implementation detail, and not such an important one at that. It's good enough that it's inserted, somewhere.